lib/cli.js

```diff
@@ -185,6 +185,36 @@ yargs
             Mcdev.retrieveAsTemplate(argv.BU, argv.TYPE, argv.NAME, argv.MARKET);
         },
     })
+    .command({
+        command: 'buildTemplate <BU> <TYPE> <KEY> <MARKET>',
+        aliases: ['bt'],
+        desc: 'builds a template out of a specific metadata file already in your retrieve folder',
+        builder: (yargs) => {
+            yargs
+                .positional('BU', {
+                    type: 'string',
+                    describe:
+                        'the business unit to deploy to (in format "credential name/BU name")',
+                })
+                .positional('TYPE', {
+                    type: 'string',
+                    describe: 'metadata type',
+                })
+                .positional('KEY', {
```

Comment:
if can accept multiple then KEYS would be better i guess :)

Reply:
Single key is default case, multiple keys the advanced case. I'd rather go with the simple default case when naming variables